src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -11,18 +11,12 @@
 import java.io.IOException;
 import java.util.*;
 import javax.annotation.Nonnull;
+import hudson.*;

```

Comment:
I am not sure what the best practice in Java is, so we should check that before importing all the symbols.